src/components/Footer/Footer.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './Footer.scss';
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <footer className="footerWraper">
+        <div className="footerNavWraper">
+          <div className="footerNavList">
+            <li className="footerNavItem">공지사항</li>
+            <li className="footerNavItem">이용약관</li>
+            <li className="footerNavItem policy">개인정보처리방침</li>
+            <li className="footerNavItem">영상정보처리방침</li>
+            <li className="footerNavItem">매장안내</li>
+            <li className="footerNavItem">고객센터</li>
+            <li className="smallSelect">
+              <select className="globalSite">
+                <option>Global Skinfood</option>
+                <option>US</option>
+                <option>Japan</option>
+                <option>Hong kong</option>
+                <option>Iraq</option>
+                <option>Turkey</option>
+                <option>Vietnam</option>
+              </select>
+            </li>
+            <li className="smallSelect">
+              <select className="language">
+                <option value="Language">Language</option>
+                <option value="Korean">Korean</option>
+                <option value="English">English</option>
+              </select>
+            </li>
+          </div>
```

Comment:
```suggestion
          <ul className="footerNavList">
            <li className="footerNavItem">공지사항</li>
            <li className="footerNavItem">이용약관</li>
            <li className="footerNavItem policy">개인정보처리방침</li>
            <li className="footerNavItem">영상정보처리방침</li>
            <li className="footerNavItem">매장안내</li>
            <li className="footerNavItem">고객센터</li>
            <li className="smallSelect">
              <select className="globalSite">
                <option>Global Skinfood</option>
                <option>US</option>
                <option>Japan</option>
                <option>Hong kong</option>
                <option>Iraq</option>
                <option>Turkey</option>
                <option>Vietnam</option>
              </select>
            </li>
            <li className="smallSelect">
              <select className="language">
                <option value="Language">Language</option>
                <option value="Korean">Korean</option>
                <option value="English">English</option>
              </select>
            </li>
          </ul>
```
리스트를 표현할 땐 꼭 ul태그로 감싸주세요!